mne_bids/mne_bids.py

```diff
@@ -72,10 +74,19 @@ def _channels_tsv(raw, fname, verbose):
         The data as MNE-Python Raw object.
     fname : str
         Filename to save the channels.tsv to.
+    write_mode : str, one of ('overwrite', 'error')
+        How the file should be handled if is exists already.
+        Defaults to 'overwrite'.
+        If overwrite = `overwrite` the old file will be removed and replaced
+        with a new one.
```

Comment:
not sure, but shouldn't this be `if overwrite=='overwrite'`

Reply:
you mean `write_mode == 'overwrite'` ?

Reply:
yes ...

Reply:
gah! How did I miss all these!? I swear I checked these things!!